prod/westeurope/internal/api/cosmosdb/container_messages/terragrunt.hcl

```diff
@@ -26,8 +26,11 @@ inputs = {
   account_name        = dependency.cosmosdb_account.outputs.name
   database_name       = dependency.cosmosdb_database.outputs.name
   partition_key_path  = "/fiscalCode"
+  throughput          = 2200
```

Comment:
I think this is too low value. We can have high user impact if we break the limit.